doc/source/whatsnew/v0.4.x.rst

```diff
@@ -1,6 +1,6 @@
 .. _whatsnew_04x:
 
-v.0.4.1 through v0.4.3 (September 25 - October 9, 2011)
+Versions 0.4.1 through 0.4.3 (September 25 - October 9, 2011)
 -------------------------------------------------------
```

Comment:
Can you make these lines the same length as the title please? Other than that looks good to me.

@TomAugspurger are you ok using `Version 1.0.3` in the what's new titles, instead of `v1.0.3`? We're applying the script to keep titles consistent in all docs, and the `v1.0.3` format will require complicating the script significantly to be handled. In some cases it'll appear at the beginning, and the script will expect `V1.0.3`, and in others it's in the middle. We can define exceptions in the script, but we should add every existing version there, and keep adding the future ones. Which seems a worse solution than just using `Version 1.0.3`.

Reply:
Version 1.0.3 seems fine.